src/core/sys/windows/winnt.d

```diff
@@ -2397,6 +2397,7 @@ struct TOKEN_OWNER {
     PSID Owner;
 }
 alias TOKEN_OWNER* PTOKEN_OWNER;
+const int SECURITY_MAX_SID_SIZE = 68;
```

Comment:
Constants should be `enum`s.

Reply:
Also this should use the appropriate type alias instead of `int`
(ditto for the other changes)

Reply:
This is a constant: https://docs.microsoft.com/en-us/windows/win32/secbiomet/general-constants

Reply:
`enum` can be used to declare compile time constants
```suggestion
enum int SECURITY_MAX_SID_SIZE = 68;
```